boards/risc-v/qemu-rv/rv-virt/scripts/ld-kernel.script

```diff
@@ -20,9 +20,9 @@
 
 MEMORY
 {
-    kflash (rx) : ORIGIN = 0x80000000, LENGTH = 2048K   /* w/ cache */
-    ksram (rwx) : ORIGIN = 0x80200000, LENGTH = 2048K   /* w/ cache */
-    pgram (rwx) : ORIGIN = 0x80400000, LENGTH = 4096K   /* w/ cache */
+    kflash (rx) : ORIGIN = 0x80000000, LENGTH = 1024K   /* w/ cache */
```

Comment:
The alignment requirement is reflected here as well. The minimum alignment for these is 4M for Sv32!